zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -157,10 +156,9 @@ object ServerSpec extends HttpRunnableSpec(8088) {
           assertM(res)(isSome(equalTo(value)))
         }
       } +
-      testM("file-streaming") {
```

Comment:
Can we keep this test Helpful for testing thing when `Blocking` is involved

Reply:
Agree! Added the test back.